tiledb/cloud/notebook.py

```diff
@@ -14,9 +27,11 @@ def rename_notebook(
 ):
     """
     Update an array's info
-    :param str namespace: optional username or organization array should be registered under. If unset will default to the user
+    :param str namespace: optional username or organization array should be
```

Comment:
`str` no longer needed since that information appears in the signature

Reply:
this is not a function i touched -- it was the function that was here when i started. and its doc string is out of data. and that info does not appear in the array signature.

i'll (try to) clean this up while i'm here, even though i didn't introduce it.